native_field/src/lib.rs

```diff
@@ -0,0 +1,266 @@
+#![no_std]
+
+extern crate alloc;
+
+use alloc::vec;
+use alloc::vec::Vec;
+use columns::{NativeFieldCols, COL_MAP, NUM_COLS};
+use core::mem::transmute;
+use valida_bus::{MachineWithGeneralBus, MachineWithRangeBus8};
+use valida_cpu::MachineWithCpuChip;
+use valida_machine::{instructions, Chip, Instruction, Interaction, Machine, Operands, Word};
+use valida_range::MachineWithRangeChip;
+
+use p3_air::VirtualPairCol;
+use p3_field::{Field, PrimeField32};
+use p3_matrix::dense::RowMajorMatrix;
+use p3_maybe_rayon::*;
+
+const ADD_OPCODE: u32 = 13;
+const SUB_OPCODE: u32 = 14;
+const MUL_OPCODE: u32 = 15;
+
+pub mod columns;
+pub mod stark;
+
+#[derive(Clone)]
+pub enum Operation {
+    Add(Word<u8>, Word<u8>, Word<u8>),
```

Comment:
nit: please comment on order, looks like dest is first? Or could do `Add { dst: Word<u8>, ... }` or what not